### DISCUSSION TITLE
Feature request: Skeleton

### DISCUSSION THREAD

**Discussion by mjsilva** [tamagui/tamagui#2185] [General]
Looks like at one point there was a Skeleton component https://github.com/tamagui/tamagui/issues/1857 but has since then been commented out.

Any plans to reintroduce it?

**Comment by fl0ydj** [comment]
Would love to see that!

**Reply by mfrfinbox** [reply]
I would love to have this too

**Comment by dylan-westbury** [comment]
anything else in the meantime that anyone has implemented?

**Reply by giacomocerquone** [reply]
just for reference, this is a very old and well maintained package https://github.com/danilowoz/react-content-loader 

That supports a lot of different frameworks. It even has a community gallery https://skeletonreact.com/#gallery
